src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -457,6 +462,59 @@ static void ConfigureFeatureFlags()
 #endif
 		}
 
+		/// <summary>
+		/// a simple best-effort parsing of CLI args as feature flags
+		/// </summary>
+		static void ParseCommandLineFeatureFlags()
+		{
+			var commandLineArgs = Environment.GetCommandLineArgs();
+			if (commandLineArgs.Length == 1)
+			{
+				return;
+			}
+
+			var availableFlags = new Dictionary<string, PropertyInfo>();
+
+			foreach (var featureClass in typeof(FeatureConfiguration).GetNestedTypes(BindingFlags.Public | BindingFlags.Static))
+			{
+				foreach (var featureProperty in featureClass.GetProperties(BindingFlags.Public | BindingFlags.Static))
+				{
+					availableFlags[$"{featureClass.Name}.{featureProperty.Name}"] = featureProperty;
+				}
+			}
+
+			var regex = new Regex(@"^--(\w+\.\w+)=(.+)$");
```

Comment:
Very nice! It's likely that the syntax will conflict with other parameters, let's prefix with `--feature-`